iphone/TitaniumKit/TitaniumKit/Sources/API/TiWindowProxy.m

```diff
@@ -505,6 +492,21 @@ - (TiProxy *)tabGroup
   return [tab tabGroup];
 }
 
+- (void)assignStatusBarStyle:(int)style
+{
+  switch (style) {
+  case UIStatusBarStyleDefault:
+  case UIStatusBarStyleLightContent:
+#if __IPHONE_OS_VERSION_MAX_ALLOWED >= 130000
+  case UIStatusBarStyleDarkContent:
```

Comment:
Doesn't this crash on devices using iOS < 13 and for example no stye? Then it will iterate the switch-cases and find undefined symbols near `UIStatusBarStyleDarkContent `.

Reply:
@vijaysingh-axway can you double check this?

Reply:
@hansemannn  I have verified it for iOS < 13. It does not crash and has default behaviour.